Text/Megaparsec/Byte/MachineInts.hs

```diff
@@ -0,0 +1,110 @@
+-- TODO debug
```

Comment:
I think we could call this module `Text.Megaparsec.Byte.Binary`.

Reply:
That's what I called it at first, then stopped and looked. Thinking about it again it makes sense as an abbreviation for "binary-format numbers". Will change